test/system/post_test.rb

```diff
@@ -0,0 +1,27 @@
+require "application_system_test_case"
+# https://guides.rubyonrails.org/testing.html#implementing-a-system-test
+
+class PostTest < ApplicationSystemTestCase
+
+  def setup
+    activate_authlogic
+  end
+
+  test 'posting from the editor' do
+    UserSession.create(users(:bob)) # log in
+
+    visit '/post'
+
+    fill_in("input#title-input", with: "My new post")
```

Comment:
You'd think this would work, but it doesn't seem to!

Reply:
Hi @jywarren , the syntax is correct . I think we are not logged-in and hence this text field is not visible .